src/app/app.component.html

```diff
@@ -1 +1,7 @@
-<router-outlet></router-outlet>
\ No newline at end of file
+<!--TODO: Inject navigation inside header component-->
+<div [ngClass]="{'light' : isDarkModeOn}">
```

Comment:
Is this functional? I dont see any function for this...

Reply:
yes it is ... we are getting the output from the header component